docs/asm-gke-terraform/main.tf

```diff
@@ -0,0 +1,39 @@
+resource "google_container_cluster" "cluster" {
+  name                    = "my-cluster"
+  location                = "us-central1-a"
```

Comment:
use `var.region` instead?

Reply:
I'd like to have the tf resource just be as minimal as possible - just plug your project id in. 
i  can change this to have a default value in `variables.tf` instead and change it to var.region? wdyt?

Reply:
Sorry I meant `var.zone`, which is already in your `variables.tf` isn't it?

Reply:
oh gotcha, yes can do!